### PR TITLE
docs: add changelog to website

### LINKED ISSUE #3143: Expose change log in the website
First of all, thank you `chakra-ui` community for this awesome library! 🤝 

# 🚀 Feature request

Combine change logs from all packages and present them in the website

## 🧱 Problem Statement / Justification

Currently to see what actually had changed, you have to click through multiple "releases" pages on GitHub. Most of the releases are "dependency update" which makes reading the actual changes quite frustrating.

## ✅ Proposed solution or API

It would be great to have a "changelog" section on the website to quickly see what were the lates changes in the components

### PR DESCRIPTION
**Pull request by estheragbaje**
Closes #3143

## 📝 Description

Updated the format and layout of the changelog.

## ⛳️ Current behavior (updates)

Got changelog to show up in the website.

## 💣 Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by with-heart** [comment]
Don't have time to review this atm but this will be so nice to have!

File: website/pages/changelog.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { parseMarkdownFile } from "@docusaurus/utils"
+import path from "path"
```
Comment (lazarnikolov94) [inline website/pages/changelog.tsx:2]: These imports are unused

File: website/pages/changelog.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import React from "react"
+import { MDXLayoutProvider } from "../layouts/mdx"
+import ChangelogContent from "../../CHANGELOG.md"
+
+// DOM code
+function Changelog({ content }) {
```
Comment (lazarnikolov94) [inline website/pages/changelog.tsx:8]: Also `content` is not being used

**Review by lazarnikolov94 — CHANGES_REQUESTED** [review]
Should we add a Changelog link somewhere on the website?

File: website/pages/changelog.tsx (hunk 2 above)
Comment (estheragbaje) [reply website/pages/changelog.tsx:8]: Thanks Lazar. Removing the unused imports.

**Pull request merged**